src/idl_gen_kotlin.cpp

```diff
@@ -1443,13 +1469,18 @@ class KotlinGenerator : public BaseGenerator {
   }
 
   // Prepend @JvmStatic to methods in companion object.
-  static void GenerateJvmStaticAnnotation(CodeWriter &code,
-                                          bool gen_jvmstatic) {
-    if (gen_jvmstatic) {
+  static void GenerateJvmStaticAnnotation(CodeWriter &code, bool generate) {
+    if (generate) {
       code += "@JvmStatic";
     }
   }
 
+  static void MarkExperimentalUnsignedTypesAnnotation(bool generate) {
```

Comment:
This must go away. Get this information at the beginning of the code generation in a field and reuse it everywhere you need it (I believe you only need it once to annotate the class).